.rubocop_todo.yml

```diff
@@ -1,15 +1,25 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2016-06-10 16:00:41 +0200 using RuboCop version 0.39.0.
+# on 2016-07-12 13:55:29 +0200 using RuboCop version 0.39.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
+# Offense count: 1
+Metrics/AbcSize:
+  Max: 18
```

Comment:
The purpose of the `.rubocop_todo.yml` file is usually not to *add more* to it ;)